halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/KayentaProfileFactory.java

```diff
@@ -108,6 +114,19 @@ protected void setProfile(Profile profile, DeploymentConfiguration deploymentCon
           enabled = googleSvc.isStackdriverEnabled();
         }
       }
+
+      @Data
```

Comment:
If this is needed for serialization/deserialization you might need to add a `@NoArgsConstructor` here since you overrode the default

Reply:
Do the profiles get deserialized somewhere (only supporting `localgit` so far, so maybe there is a path)?